.gitignore

```diff
@@ -3,5 +3,6 @@ public/lib
 .sass-cache/
 .DS_store
 c4h_keys.txt
+./test/unit
```

Comment:
You asked me to remove my test file inside the test/unit folder, I have to ignore it since the folder is now empty or should I remove the the ./test/unit from gitignore?